lib/fog/aws.rb

```diff
@@ -219,7 +221,7 @@ def self.parse_security_group_options(group_name, options)
 
     def self.json_response?(response)
       return false unless response && response.headers
-      response.get_header('Content-Type') =~ %r{application/json}i ? true : false
+      response.get_header('Content-Type') =~ %r{application/.*json.*}i ? true : false
```

Comment:
@geemus kinesis has a new content type:

`x-amz-json-1.1`. Actually it looks like data_pipeline and dynamodb use a similar content type, but apparently this is maybe the first time `json_response?` was need to support error parsing. Does this new match look ok?